### PR TITLE
Fix to bug with opening multiple modals.

### PR DESCRIPTION
**Pull request by brittainhard**

### CODE REVIEW

**Comment by brittainhard** [comment]
@ahmadia after this has been merged i think it's safe to cut a release.

**Comment by ahmadia** [comment]
This feels a little bit fragile, but I don't know enough JS to come up with a better solution.

**Comment by ahmadia** [comment]
I'm fine with you merging this in, but I can't provide a good review for you here.

**Comment by brittainhard** [comment]
@ahmadia I got the solution from here: https://github.com/nakupanda/bootstrap3-dialog/issues/70

File: source/base/static/base/js/backbone/crawl_models.js [hunk 1]
```diff
@@ -92,6 +92,15 @@
       var that = this;
       this.collection = collection;
       this.collectionView = collectionView;
+
+      // If the add crawl modal has been opened at the same time as the add model form,
+      // change focus back to the add crawl modal after the form has been closed
+      // or submitted.
+      $(this.modal).on('hidden.bs.modal', function(event){
+        if ($("#addCrawlModal").attr("aria-hidden") == "false"){
```
Comment (ahmadia) [inline source/base/static/base/js/backbone/crawl_models.js]: Why do we need this additional check?

Comment (brittainhard) [inline source/base/static/base/js/backbone/crawl_models.js]: Basically, what happens is that when you close a modal, it removes the `modal-open` class from the body. If there is another modal open, then it breaks the scrolling for modal that is still open.

If the add crawl modal is open, then we want to re-add that `modal-open` class to make sure that the scrolling doesn't break.

We only want this to happen if the add crawl modal is open, hsnce the check for "aria-hidden"

Comment (ahmadia) [inline source/base/static/base/js/backbone/crawl_models.js]: okay, I understand more of what's going on here.  There are more general solutions to this problem, but they are probably unnecessary complications.  Let's remember to come back and refactor this if we end up needing this in more than one or two places.

**Pull request merged**